Userland/Libraries/LibCrypto/Curves/X448.cpp

```diff
@@ -0,0 +1,356 @@
+/*
+ * Copyright (c) 2022, stelar7 <dudedbz@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/ByteReader.h>
+#include <AK/Endian.h>
+#include <LibCrypto/Curves/X448.h>
+
```

Comment:
I feel like most of this code can be shared with the other curve, maybe with one or two template parameters?
Not a big problem, but it'd be nice to not repeat so much (if it is in fact repeated).

Reply:
There are some repeated methods (import, export, copy, select, set), but the "main" methods are unique in the fact that they do a bit extra work, and have some different constants. It could all be combined with some `if constexpr` stuff, but I would prefer if it wasnt done, since its more readable this way

Reply:
That seems alright then, no need to make a constexpr mess